tests/integration/targets/mso_rest/tasks/error_handling.yml

```diff
@@ -82,7 +82,7 @@
     that:
     - error_on_name_resolution.msg is search("Name or service not known")
   when: 
-    - version.current.version is version('3.7', '>=')
+    - version.current.version is version('3.7', '>=') and version.current.version is version('4.1', '<')
```

Comment:
Is the error message different in 4.1+?

Reply:
I added a condition to avoid the test for the current NDO because the `host` is `foo.bar.cisco.com`. It won't work forever, but someone added the assertion task.

Reply:
But that is the goal of this test is to trigger an error and check the error message right?

Reply:
I get the error message below, looks like the error message change is because of the latest ND HTTPAPI and Ansible. Shall I remove task from `tests/integration/targets/mso_rest/tasks/error_handling.yml`?

```
ND HTTPAPI send_request() Exception: Could not connect to https://foo.bar.cisco.com:443/mso/api/v1/tenants: [Errno 8] nodename nor servname provided, or not known
```

Reply:
Removed the version check from the assertion task because the statement is applicable to the Linux environment.